tests/foreman/ui_airgun/test_lifecycleenvironment.py

```diff
@@ -0,0 +1,66 @@
+"""Test class for Lifecycle Environment UI
+
+:Requirement: Lifecycleenvironment
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: UI
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+from nailgun import entities
+
+
+from robottelo.datafactory import gen_string
+from robottelo.decorators import fixture, tier2
+
+
+@fixture(scope='module')
+def module_org():
+    return entities.Organization().create()
+
+
+def test_positive_edit(session):
+    lce_path_name = gen_string('alpha')
+    new_name = gen_string('alpha')
+    with session:
+        session.lifecycleenvironment.create(
+            values={'name': lce_path_name}
```

Comment:
imo explicitly specifying arg names is not necessary here. but that's definitely nitpick